modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java

```diff
@@ -1177,6 +1178,13 @@ private void _addObjectDefinitions(
 				"OBJECT_DEFINITION_CLASS_NAME:" +
 					serviceBuilderObjectDefinition.getShortName(),
 				serviceBuilderObjectDefinition.getClassName());
+			stringUtilReplaceValues.put(
+				"OBJECT_DEFINITION_PORTLET_ID:" +
+					serviceBuilderObjectDefinition.getShortName(),
+				ObjectPortletKeys.OBJECT_DEFINITIONS +
+					StringUtil.split(
+						serviceBuilderObjectDefinition.getClassName(),
+						StringPool.POUND)[1]);
 			stringUtilReplaceValues.put(
 				"OBJECT_DEFINITION_ID:" +
 					serviceBuilderObjectDefinition.getShortName(),
```

Comment:
@vngoliveira `OBJECT_DEFINITION_PORTLET_ID` should be after `OBJECT_DEFINITION_ID` to follow alphabetical order